packages/makeup-hoist-element/README.md

```diff
@@ -0,0 +1,39 @@
+# makeup-hoist-element
+
+The makeup-hoist-element `hoist` function Takes an element in the DOM and hoists it as a direct child under the `document.body`.
+
+The `unhoist` function puts the element back to its previous location. 
+
+This helps with things like screenreader traps and accessibility. 
+
+## Experimental
+
+This CommonJS module is still in an experimental state, until it reaches v1.0.0 you must consider all minor releases as breaking changes. Patch releases may introduce new features, but will be backwards compatible.
+
+## Demo
+
+See this in use with the makeup-modal demo.
+
+```js 
+const hoist = require('makeup-hoist-element')
+
+// hoist element
+hoist.hoist(document.querySelector('el'));
+
+// unhoist element
+hoist.unHoist();
```

Comment:
So I think you are copying the screenreader-trap API here, which is a singleton (makes sense as there can only be a single trap on screen at any time). Are we also saying that there can only be one hoisted element at at time? Something to think about.

Reply:
Yeah in this case keeping things in order would be fairly complicated if you were hoisting more than one thing at a time. 
I think a singleton pattern makes sense.